README.md

```diff
@@ -455,6 +459,41 @@ register('select-url', URLOperation);
 register('report', ReportOperation);
 ```
 
+### Using cross-origin worklets
```

Comment:
This section is a bit confusing, as it's using cross-origin to mean different things. Use cases 1 and 2 are about cross-origin worklet script. Use case 3 is about a cross-origin data origin.

Can you instead make 2 sections, one titled "Loading cross-origin worklet scripts" and another, "Creating worklets with cross-origin data origins" or some such?

Reply:
I changed the section title to "Loading cross-origin worklet scripts" and slightly revised the content. 

I think that it would make more sense to wait to add a separate "Creating worklets with cross-origin data origins" section until we decide to allow other values for the `dataOrigin` option (if we do).

Anyway, please take a look and see if the revised section looks good. If you would still like it in 2 sections, I can split it then. Thanks.

Reply:
| I think that it would make more sense to wait to add a separate "Creating worklets with cross-origin data origins" section until we decide to allow other values for the dataOrigin option (if we do).

Using `script-origin` _is_ a cross-origin data origin from the calling context.

Reply:
So does the revised section look good for the current proposal? Or are there any changes that we should make in this PR?